src/main/jbake/content/admin/index.adoc

```diff
@@ -1,9 +0,0 @@
-= TomEE Administration
```

Comment:
do we loose the page? If somebody bookmarked http://tomee.apache.org/admin/ is it a 404 now?